ota-plus-web/conf/logback.xml

```diff
@@ -1,3 +1,15 @@
 <configuration>
   <include resource="logback-libats.xml"/>
+
```

Comment:
Why is it needed? Is not the include handles it?

Reply:
Unfortunately, no. I had to create a distinct appender with `msgIsJson` option set. Otherwise, JSON messages will be backslashed.

Reply:
Could you please doublecheck? it is what the rest of the services are using.